test/ess-tests.el

```diff
@@ -32,4 +29,4 @@
   (should (string= 'ess-julia-mode (with-temp-buffer (ess-julia-mode) major-mode))))
 
 (ert-deftest sas-mode ()
-  (should (string= 'ess-mode (with-temp-buffer (SAS-mode) major-mode))))
+  (should (string= 'SAS-mode (with-temp-buffer (SAS-mode) major-mode))))
```

Comment:
Probably should be `sas-mode`?

Reply:
There are weird functions that convert the string ess-dialect into a symbol and call that symbol in order to start the inferior mode so we'll at least have to provide SAS-mode as an alias.

Reply:
I see. Do as easiest, it's not that important.